src/Cogworks.AzureSearch.IoC.Umbraco/Cogworks.AzureSearch.IoC.Umbraco.csproj

```diff
@@ -15,18 +15,18 @@
         <None Include="..\..\README.md" Pack="true" PackagePath="\" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
+        <PackageReference Include="Umbraco.Cms.Core" Version="[13.3.2,)" />
```

Comment:
Please make from 13.0.0 as begging version

Reply:
I tried actually but pipelines gave an error: https://github.com/thecogworks/Cogworks.AzureSearch.IoC.Umbraco/actions/runs/9417358219/job/25942374104
![image](https://github.com/thecogworks/Cogworks.AzureSearch.IoC.Umbraco/assets/1363876/ca4be304-3f4f-469a-b62a-73dfa86078d6)

Reply:
https://github.com/advisories/GHSA-74p6-39f2-23v3

Reply:
You can make the `NU1902` as a warning.

We can still say minimal Umbraco version and it's up to the project configuration to choose the current Umbraco version 🤔